swap.c

```diff
@@ -0,0 +1,28 @@
+#include<stdio.h>
+
+void callByValue(int a, int b) {
+	int temp;
+	temp = a;
+	a = b;
+	b = temp;
+}
+
+void callByReference(int *a, int *b) {
+	int temp;
+	temp = *a;
+	*a = *b;
+	*b = temp;
+}
+
+int main() {
+    int a, b;
+    printf("Enter the element of a and b:");
```

Comment:
Please follow this on every printf statements for better formatting
```suggestion
    printf("Enter the element of a and b:\n");
```

Reply:
ok, I will follow.